playground-xyz-htb.js

```diff
@@ -290,10 +389,19 @@ function PlaygroundXyzHtb(configs) {
                  */
 
                 /* ----------- Fill this out to find a matching bid for the current parcel ------------- */
-                if (curReturnParcel.xSlotRef.someCriteria === bids[i].someCriteria) {
-                    curBid = bids[i];
-                    bids.splice(i, 1);
-                    break;
+
+                if (!curReturnParcel.xSlotRef.sizes) {
```

Comment:
This if clause is not really necessary, as the validator file will ensure the sizes always exist in the config. Please remove.